survey/mutations.py

```diff
@@ -80,6 +82,9 @@ def mutate(self, info, data):
             boundary=data.boundary,
         )
         if data.attachment:
-            survey.attachment.add(*data.attachment)
+            for file in data.attachment:
+                gallery = Gallery(media=file, title=file.name, type="image")
```

Comment:
Is it guaranteed uploaded file will be of image type? If there is no guarantee then add validation to verify the media file. Can be added to Galley model also for all media types